liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -1174,7 +1178,8 @@ private void libertyDependencyWarning(boolean generateFeatures, Exception e) {
         public void installFeatures(File configFile, File serverDir, boolean generateFeatures) {
             try {
                 ServerFeatureUtil servUtil = getServerFeatureUtil(true, libertyDirPropertyFiles);
-                Set<String> features = servUtil.getServerFeatures(serverDir, libertyDirPropertyFiles);
+                FeaturesPlatforms fp = servUtil.getServerFeatures(serverDir, libertyDirPropertyFiles);
+                Set<String> features = fp.getFeatures();
```

Comment:
Seeing the configuration features get passed below to the install-feature mojo made me think about how we said versionless features would not be supported in pom.xml/build.gradle. It seems like some validation should be done to detect a feature without a version and throw an error (or at least log a warning/error).

Reply:
Agree

Reply:
Opened issue to add some tests so we can close on this PR.